src/java.base/share/native/libnet/net_util.c

```diff
@@ -74,7 +74,7 @@ DEF_JNI_OnLoad(JavaVM *vm, void *reserved)
     s = (*env)->NewStringUTF(env, "java.net.preferIPv4Stack");
     CHECK_NULL_RETURN(s, JNI_VERSION_1_2);
     preferIPv4Stack = (*env)->CallStaticBooleanMethod(env, iCls, mid, s);
-
+    JNU_CHECK_EXCEPTION_RETURN(env, JNI_VERSION_1_2);
```

Comment:
The exception check is technically not necessary because no other JNI methods are called before the function exits, but it wasn't always the case.

Reply:
just so we don't forget to add it when we modify the enclosing function. The call to `CallStaticBooleanMethod` is located pretty far from the end of the function, and can be easy to miss.

Reply:
yes, at least on Solaris we used to call other JNI methods after this one, see `parseExclusiveBindProperty` before 071bd521bca2485c6df95c45e4310d39b05bd046